src/main/resources/db/migrations/V1__CreateTables.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE IF NOT EXISTS User(
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS Users(
```

`User` is a reserved word in PostgreSQL.